examples/test_xworld_dialog.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018 Baidu Inc. All Rights Reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "simulator_interface.h"
+#include "games/xworld/xworld_simulator.h"
+
+using namespace simulator::xwd;
+using namespace simulator;
+
+DECLARE_int32(context);
+DECLARE_bool(task_groups_exclusive);
+DECLARE_string(task_mode);
+DECLARE_bool(pause_screen);
+
+void game_reset_with_teacher(SimulatorPtr game, TeacherPtr teacher) {
```

Comment:
this function is not being used? plus xwd now has teacher inside

Reply:
Removed.